src/sed.js

```diff
@@ -32,19 +33,32 @@ function _sed(options, regex, replacement, file) {
   if (typeof regex === 'string')
     regex = RegExp(regex);
 
-  if (!file)
-    common.error('no file given');
+  if (!files)
+    common.error('no paths given');
 
-  if (!fs.existsSync(file))
-    common.error('no such file or directory: ' + file);
+  if (typeof files === 'string')
+    files = [].slice.call(arguments, 3);
+  // if it's array leave it as it is
 
-  var result = fs.readFileSync(file, 'utf8').split('\n').map(function (line) {
-    return line.replace(regex, replacement);
-  }).join('\n');
+  files = common.expand(files);
 
-  if (options.inplace)
-    fs.writeFileSync(file, result, 'utf8');
+  var sed = [];
+  files.forEach(function(file) {
+    if (!fs.existsSync(file)) {
```

Comment:
The current `expand()` will not filter out nonexistent files, so I think this should still be in place. For example:

```Javascript
> sed('search', 'replacement', 'nonexistentfilename');
sed: no such file or directory: nonexistentfilename
```